purchase_request_to_rfq/models/purchase_request.py

```diff
@@ -80,50 +80,44 @@ def _planned_date(self, request_line, delay=0.0):
             or False
 
     @api.model
-    def _calc_new_qty_price(self, request_line, po_line=None, cancel=False):
-        uom_obj = self.env['product.uom']
-        qty = uom_obj._compute_qty(request_line.product_uom_id.id,
-                                   request_line.product_qty,
-                                   request_line.product_id.uom_po_id.id)
+    def _get_supplier_min_qty(self, product, partner_id=False):
+        seller_min_qty = 0.0
+        if partner_id:
+            seller = product.seller_ids \
+                .filtered(lambda r: r.name == partner_id) \
+                .sorted(key=lambda r: r.min_qty)
+        else:
+            seller = product.seller_ids.sorted(key=lambda r: r.min_qty)
+        if seller:
+            seller_min_qty = seller[0].min_qty
+        return seller_min_qty
+
+    @api.model
+    def _calc_new_qty(self, request_line, po_line=None, cancel=False,
+                      new_pr_line=False):
+        uom = request_line.product_uom_id
+        qty = uom._compute_qty(
+            request_line.product_id.uom_po_id, request_line.product_qty)
```

Comment:
When porting this to v10, I realized that the `_compute_qty()` method is never with the correct inputs... [see definition](https://github.com/odoo/odoo/blob/9.0/addons/product/product.py#L107).